doc/source/cluster/cluster_under_construction/running-applications-on-ray-clusters/job-submission/index.md

```diff
@@ -1,28 +1,41 @@
-(ref-deployment-guide-under-construction)=
+(ray-jobs-under-construction)=
 
-# Job Submission
+# Ray Jobs
 :::{warning}
 This page is under construction!
 :::
 
+Once you have deployed a Ray cluster (on [VMs](ref-cluster-quick-start-vms-under-construction) or [Kubernetes](kuberay-quickstart)), you are ready to run a Ray application!
+
+The recommended way to run a Ray application on a Ray Cluster is to use *Ray Jobs*.
+This mechanism allows you to submit locally developed applications to a remote Ray Cluster for execution.
+It simplifies the experience of packaging, deploying, and managing a Ray application.
+
 :::{note}
-(cade@) This was migrated from the old deployment guide
+This component is in **beta**. APIs may change before becoming stable.
 :::
 
-This section explains how to set up a distributed Ray cluster and run your workloads on it.
+A Ray Job consists of:
+1. An entrypoint command, like `python my_script.py`.
+2. A [runtime environment](runtime-environments), which specifies the application's file and package dependencies.
 
-To set up your cluster, check out the {ref}`Ray Cluster Overview <cluster-index>`, or jump to the {ref}`Ray Cluster Quick Start <ref-cluster-quick-start>`.
+A Ray Job can be submitted by a remote client that lives outside of the Ray Cluster, which we will show in the following user guides.
```

Comment:
grammar nit on dangling `, which`.
Consider splitting into two sentences e.g.
`A Ray Job can be submitted by a remote client that lives outside of the Ray Cluster; we will show this workflow in the following user guides.`